### PR TITLE
Test

### PR DESCRIPTION
**Pull request by looh16**
Create unit tests for all components.

### CODE REVIEW

File: src/test/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly Calculator Component ', () => {
+  const tree = Renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (medaminedev66) [inline src/test/Calculator.test.js:10]: - [ ] Following the project requirements, you should use also React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @looh16 and @businesskaniki  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional.
- Jest snapshots have been used for testing all the components.
- There are no linter errors.
- The tests passed successfully.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @looh16 and @businesskaniki, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 
- All tests passed 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**